api/src/main/java/org/openmrs/module/pihcore/config/ConfigDescriptor.java

```diff
@@ -0,0 +1,59 @@
+package org.openmrs.module.pihcore.config;
+
+import org.codehaus.jackson.annotate.JsonProperty;
+
+import java.util.List;
+
+/**
+ * Object that encapsulates the options that can be configured on a per-installation basis
+ */
+public class ConfigDescriptor {
+
+    public enum Site {
+        MIREBALAIS, LACOLLINE
```

Comment:
I'm not sure how scalable it is to explicitly list out all our installations here, but it's not a blocker at this point.